05 - coder-files/src/components/Button.jsx

```diff
@@ -0,0 +1,12 @@
+export default function Button({ position, children, onClick }) {
+  const handleClick = () => onClick(position);
```

Comment:
Hey @riyoneri don't you think you are repeating yourself by creating 2 `handleClick()` functions in both the child and parent Component, while you were supposed to passe it child to use it there, It could be better that you declare it in the parent component and pass it to child component